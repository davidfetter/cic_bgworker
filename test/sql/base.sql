\set ECHO none
BEGIN;
\i sql/cic_bgworker.sql
\set ECHO all

-- You should write your tests

SELECT cic_bgworker('test');

ROLLBACK;
