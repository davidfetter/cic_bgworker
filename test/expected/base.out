\set ECHO none
-- You should write your tests
SELECT cic_bgworker('test');
 cic_bgworker 
--------------
 
(1 row)

ROLLBACK;
